gui/styles/_wizard.styl

```diff
@@ -81,27 +84,83 @@ a.more-info
   color var(--slateGrey)
 
 
-.folder__selector
-  @extend $form-select select
-  width: 100%
-  font-size: 1.1em
-  line-height: 1em
-  margin 0
-  text-align left
-  text-decoration: none
+.folder__subtitle
+  margin-bottom 0
+
+.folder__config
+  margin-top 1rem
+
+  ul
+    margin-bottom 0
+    padding-inline-start: 1rem;
```

Comment:
It is a bit weird to have `padding-inline-start` here, and `padding-left`, `margin-left`, etc. in other places.

Reply:
I'll check whether `padding-left` does the same or not.